test/views/ThirdIndustryOptionsViewSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import forms.ThirdIndustryOptionsFormProvider
+import models.{NormalMode, ThirdIndustryOptions}
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.ThirdIndustryOptionsView
+
+class ThirdIndustryOptionsViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "thirdIndustryOptions"
+
+  val form = new ThirdIndustryOptionsFormProvider()()
+
+  val application = applicationBuilder(userAnswers = Some(emptyUserAnswers)).build()
+
+  val view = application.injector.instanceOf[ThirdIndustryOptionsView]
+
+  def applyView(form: Form[_]): HtmlFormat.Appendable =
+    view.apply(form, NormalMode)(fakeRequest, messages)
+
+  "ThirdIndustryOptionsView" must {
+
+    behave like normalPage(applyView(form), messageKeyPrefix)
+
+    behave like pageWithBackLink(applyView(form))
+  }
+
+  "ThirdIndustryOptionsView" when {
+
+    "rendered" must {
+
+      "contain radio buttons for the value" in {
+
+        val doc = asDocument(applyView(form))
+
+        for (option <- ThirdIndustryOptions.options) {
+          assertContainsRadioButton(doc, option.id, "value", option.value, false)
+        }
+      }
+    }
+
+    for (option <- ThirdIndustryOptions.options) {
```

Comment:
Noticed that this is repeated a few times, would it not be worth making a behaviour?